packages/mui-base/src/FormControlUnstyled/index.ts

```diff
@@ -1,6 +1,7 @@
 export { default } from './FormControlUnstyled';
 
 export { default as FormControlUnstyledContext } from './FormControlContext';
+export type { FormControlUnstyledState } from './FormControlContext';
```

Comment:
@michaldudak tagged you as another reviewer since I update this export from base.

Reply:
Actually, it can be `export * from './FormControlContext';`

Reply:
Alright.